docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
+description: "Learn about the Android SDK's release channels and choose the right one."
+sidebar_order: 7000
+---
+
+The Android SDK uses the following channels to release SDK updates:
+
+## `Pre-release`
+
+Pre-releases (alpha, beta, RC,…) are for larger and potentially more impactful changes, such as new features or major versions.
+
+## `Latest`
+
+We continuously release major/minor/hotfix versions from the `main` branch. These releases go through all our internal quality gates and are very safe to use. They're intended to be the default for most teams.
+
+## `Stable`
+
+Releases are promoted from `Latest` to `Stable` when they've been used in the field for some time and at some scale. The length of time a release has been out, the level of adoption, and other quality and stability metrics are considered. These releases will be indicated on the releases page (https://github.com/getsentry/sentry-cocoa/releases/) with the `Stable` label.
+
+## FAQ
+
+### Where can I find the latest stable release for any given SDK?
+
+On the releases pages of the SDK's GitHub repo. Stable releases will be indicated with the label `(Stable)` in the name of the release. You can also use the search box in the top right to search for the newest releases.
+
+### Does this mean the release channels are not stable?
+
+No. Stability and performance are the top priorities for every release. The `Stable` release status means that the release has been tested and proven to work in the wild and at scale.
+
+### How often do we mark SDKs as stable?
+
+The target cadence is to provide stable releases every month.
```

Comment:
Maybe we shouldn't even mention how often we mark them as stable. We could have situations where we mark a release as stable every week and perhaps only every two months.

Reply:
resolving this convo in favor of https://github.com/getsentry/sentry-docs/pull/11506#discussion_r1791832673

Reply:
see https://github.com/getsentry/sentry-docs/pull/11506#discussion_r1792180999
I believe it's good to give a rough indication of what the plan is

Reply:
> What about Mobile SDK Release Channels? Or are the other SDKs also applying this process?

see https://github.com/getsentry/sentry-docs/pull/11506#discussion_r1792158408

Reply:
Maybe we should weaken the sentence by changing it to 
`The target cadence is to provide stable releases roughly every month.`

I'm afraid people will expect a stable release every month and start complaining when there is none. I'm more a fan of it's ready when it's ready. I'm still for removing that part.

Reply:
reworded to `The stable release cycle depends on a number of factors and can vary, but the target cadence is to provide stable releases roughly every month.` to manage expectations. IMO it's valuable to have this here. if it creates confusion, we'll iterate on the wording (or the process)